sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -214,16 +218,37 @@ def _from_amqp_message(self, message):
 
     def _to_outgoing_amqp_message(self):
         message_header = None
+        ttl_set = False
         if self.header:
             message_header = uamqp.message.MessageHeader()
             message_header.delivery_count = self.header.delivery_count
             message_header.time_to_live = self.header.time_to_live
             message_header.first_acquirer = self.header.first_acquirer
             message_header.durable = self.header.durable
             message_header.priority = self.header.priority
+            if self.header.time_to_live and self.header.time_to_live != MAX_DURATION_VALUE:
+                ttl_set = True
+                creation_time_from_ttl = int(time.mktime(datetime.now(UTC()).timetuple()))
+                absolute_expiry_time_from_ttl = int(min(
+                    MAX_ABSOLUTE_EXPIRY_TIME,
+                    creation_time_from_ttl + self.header.time_to_live
+                ))
+            else:
+                ttl_set = False
```

Comment:
I think the else statement is redundant?

Reply:
you're right 🤦